Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2723 -- Update "about" page and remove criteria PDF

Resolves [EREGCSC-2723](https://jiraent.cms.gov/browse/EREGCSC-2723)

**Description**

**This pull request changes:**

- Updates About page text and links per Google Doc linked in story
- Removes criteria PDF file from codebase
   
**Steps to manually verify this change:**

1. Green check marks
2. About page copy text and links on the [experimental deployment](https://1lr5qdzj7d.execute-api.us-east-1.amazonaws.com/dev1357/about/) are validated as correct
3. [Criteria for eRegulations Content Placement 11.18.21.pdf](https://dmjxios7223qb.cloudfront.net/pdfs/Criteria%20for%20eRegulations%20Content%20Placement%2011.18.21.pdf) file is no longer available to be downloaded on the experimental deployment
